cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -2229,37 +2229,38 @@
             },
             "config": {
               "type": "object",
-              "description": "Configuration settings or override Ubuntu Advantage config. Only used in non Ubuntu Pro instances.",
+              "description": "Configuration settings or override Ubuntu Advantage config.",
+              "additionalProperties": false,
```

Comment:
I actually think we want this to be true to allow for flexibility since UA currently has more config options than cloud-inti schema reports (since we only wanted to validate URL *proxy type keys). We also don't want to disallow valid confg keys as UA grows more because that would require cloud-init to continue to SRU to enable any new config keys that are supported.

Reply:
done